### DISCUSSION TITLE
Change emulator core from GLO

### DISCUSSION THREAD

**Discussion by mitchchn** [OnionUI/Onion#802] [Ideas]
### Checklist

- [X] I am reporting a bug in the Onion OS (Note that Onion devs can’t fix inherited issues with RA Cores)
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.2-beta

### Miyoo Firmware Version

Latest

### Provide a clear and concise description of the issue

It should be possible to change the default core for the entire emulator, not just the selected game.

### Steps to Reproduce

- Go to a list of games and press Y to open GLO
- There is an option for "Game core" but none for "Emulator core".

### Provide any additional context or information that may help us investigate your issue

GLO already includes options for the entire list so it's reasonable for it to have a setting like this.

When deciding on which core to use for launching a game, the  order of precedence would be: Game core > Emulator core > Default core.

**Comment by Aemiii91** [comment]
This is not a bug. Feature requests belong in discussions.
